docs/overrides/stylesheets/style.css

```
@@ -68,57 +65,35 @@ div.highlight {
   flex-direction: column;
 }
 .banner-wrapper {
-  padding: 40px 16px;
+  padding: 40px 0px;
```

💭 **SUGGESTION**: On small screens the new `padding: 40px 0px;` may cause the banner content to sit flush against the viewport edges, which can feel cramped and harder to scan. Keeping horizontal padding consistent with desktop improves readability.

**Suggested change:**
```suggestion
  padding: 40px 16px;
```